### ISSUE TITLE
Accessibility suggestions

### ISSUE THREAD

**Issue by roelvangils** [godbout/Wooshy.docs#73]
I have a few more suggestion for the "VoiceOver" target (the one I use all the time).

<img width="396" alt="VoiceOver selected" src="https://user-images.githubusercontent.com/103744/189501894-7423cfe1-e737-4d4f-bccd-92b721de21ac.png">

1. Maybe calling it VoiceOver is not the best idea. It may sound confusing to some users. It's just the target indicator that mimicks the style of the VoiceOver cursor. What about calling it "High-contrast" or "High-contrast (inspired by VoiceOver cursor)"?
2. When tabbing through the found targets, the active one gets a colour overlay. To some people who are visually impaired (including me), that might be not clear enough (depending on the situation). For the high-contrast target indicator, it might make more sense to invert the content within the target indicator. That way, it is always very clear which is being focused. 
3. Similarly to VoiceOver on the Mac, maybe you can consider adding "target magnification" so it becomes even easier to see which one is being focused (as an option).
 
![20220910_231605_VoiceOver Utility@2x](https://user-images.githubusercontent.com/103744/189502017-ac0b9fbf-2321-4ba2-bb6c-7733b1b53925.png)

Together with the "spacebar to smart zoom" option that is being considered, Wooshy could really become an very useful accessibility utility for people who have trouble using the mouse, have trouble seeing etc. Maybe you could also market it like that 😉

**Comment by godbout** [comment]
> 2\. For the high-contrast target indicator, it might make more sense to _colour-invert_ the content within the target indicator.

do you mean color-inverting Wooshy's Primary Target colors? or the window behind? because i don't think i can color-invert the window behind.

**Comment by roelvangils** [comment]
No, not the window behind :) Not Wooshy's target colour either, but the actual content of what is inside of the target. So this could (as an option) replace the target color overlay. Would you like me to create a mockup to illustrate what I mean?

**Comment by godbout** [comment]
yeah i'm gonna need a nice drawing 😅️ thanks.

**Comment by godbout** [comment]
@roelvangils drawing? coz no idea what you mean. or close if not needed anymore?

**Comment by roelvangils** [comment]
I completely forgot about this myself, but now that I’ve re-read my original request, I understand what I meant. I still believe it would be a useful feature. I would still like to create a mockup in Figma to illustrate my idea. Could you please keep this issue open a bit longer? 🙂 Like a few weeks? No time this Christmas 😕 I’ve added it to my to-do list to create the mockup so you can see what I mean. Then you can decide if it’s useful or not and close this thread.

**Comment by godbout** [comment]
yes. no worries my friend. i'm just very excited to get more info so that i can start looking into making Wooshy better! if you're aware now it's all good. no rush. Merry Christmas my favorite Belgian gay friend. am i allowed to say this? HHAHAHAHHAHHHAHH

**Comment by roelvangils** [comment]
> Merry Christmas my favorite Belgian gay friend. am i allowed to say this? HHAHAHAHHAHHHAHH

Yeah, sure 😁 Merry Christmas to you, too, you wonderfully strange (but kind) man! 😁
